osu.Game/Screens/OnlinePlay/Multiplayer/Match/Playlist/MultiplayerPlaylist.cs

```diff
@@ -121,7 +121,11 @@ protected override void PlaylistItemChanged(MultiplayerPlaylistItem item)
 
         private void addItemToLists(MultiplayerPlaylistItem item)
         {
-            var apiItem = Playlist.Single(i => i.ID == item.ID);
+            var apiItem = Playlist.SingleOrDefault(i => i.ID == item.ID);
+
+            // Item could have been removed from the playlist while the local player was in gameplay.
```

Comment:
I wonder if this means the room i was in removed the disco prince i queued up. I will never know, but I will also be sad if that was the case..